src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -90,17 +90,22 @@ Status ServiceBasedJobInfoAccessor::AsyncSubscribeAll(
 
 void ServiceBasedJobInfoAccessor::AsyncResubscribe(bool is_pubsub_server_restarted) {
   RAY_LOG(DEBUG) << "Reestablishing subscription for job info.";
+  auto fetch_all_done = [](const Status &status) {
+    RAY_LOG(INFO) << "Finished fetching all jobs from gcs server.";
```

Comment:
Can you improve logs a little bit more? Or change the log level to DEBUG

Reply:
I have improve it, thanks.